source/guides/clear/mixer.rst

```diff
@@ -411,27 +481,191 @@ mix version 10 (from which the image was built), to mix version 20.
       swupd update
       swupd bundle-list -a
 
-#. Now your mix should be at version 20 and curl is now available. Try using
+#. Now your mix should be at version 20 and curl is available. Try using
    curl. This will fail because curl is not yet installed:
 
    .. code-block:: console
 
       curl: command not found
       To install curl use: swupd bundle-add curl
 
-   Add the new bundle from your update server to your VM. Retry curl. It works!
+#. Add the new bundle from your update server to your VM. Retry curl.
+   It works!
 
    .. code-block:: bash
 
       swupd bundle-add curl
       curl -O https://download.clearlinux.org/image/start_qemu.sh
 
-   Shutdown your VM:
+#. Shutdown your VM:
 
    .. code-block:: bash
 
       poweroff
 
+Example 6: Remove a bundle from client system
+=============================================
+
+Removing a bundle in a future release requires more steps than deleting the
+bundle description file, as shown in Example 2. After a bundle is built in
+the mix, you must assure all of the files that are part of the bundle are
+removed from the client where that bundle is installed. To do this, create a
+version of this bundle in which all of its content is marked for deletion.
+
+In the following example, we show how to remove the contents of the `editors`
+bundle that we added to our mix in Example 2.
+
+#. First update your mix version. This will set the mix to the next version.
+
+   .. code-block:: bash
+
+      mixer versions update
+
+   .. note::
+      Run this command every time that you want to build a new version.
+
+#. Navigate to local-bundles:
+
+   .. code-block:: bash
+
+      cd local-bundles
+
+#. Open the `editors` bundle with an editor and delete
+   **all lines** that follow after the `[MAINTAINERS]` line.
+
+#. Afterward, it should look like this:
+
+   .. code-block:: console
+
+      # [TITLE]: editors
+      # [DESCRIPTION]: Run popular terminal text editors.
+      # [STATUS]: Active
+      # [CAPABILITIES]:
+      # [TAGS]: Tools and Utilities, Editor
+      # [MAINTAINER]: Developer <developer@intel.com>
+
+#. Save and exit.
+
+#. Next, run a build to capture recently edited bundles and update your mix.
+
+   .. code-block:: bash
+
+      mixer build all
+
+   .. note::
+      :command:`mixer build all` runs both :command:`mixer build bundles` and :command:`mixer build update` in one step.
+
+At this point the new mix, version 30, is complete. All the content of the
+editors bundles is marked as deleted. If any clients of this mix upgraded to
+mix build version 30, the content of the editors bundle would be removed.
+Note that the bundle still exists and is being tracked by swupd, but it
+contains no files.
+
+Example 7: Execute a format bump
+================================
+
+As a maintainer of your mix, you must execute a format bump if you wish to:
+* Track upstream’s format bump on your downstream derivative
```

Comment:
Lines 568 and 569 should be tabbed inward to make the bullet listing look right.

Reply:
Nope. They're correct in Sphinx's world.

Reply:
Oops. Corrected. Needed one empty line.